documentation/service_onboarding.md

```diff
@@ -0,0 +1,321 @@
+# Developer Set Up
```

Comment:
This looks very similar to `developer_setup.md`, is that expected?

Reply:
For now I'm going to get rid of this portion and add it back when it's deemed necessary.